src/molecules/ProfileSettingsForm/ProfileFormFieldsets.tsx

```diff
@@ -20,15 +23,15 @@ import {
 import { formatDate, formatPhone } from 'atoms/ProfileSettingInputs/utils';
 
 export interface IFieldsetProps {
-  getValue: (value: string) => void,
-  isValid: boolean,
+  getValue: (id: ProfileDataTypes, value: string) => void,
+  isValid: ProfileDataValidationTypes,
```

Comment:
Can't get clearly what happens here, but you can probably make isValid as:
```
isValid: {
   [keys]: boolean;
}
```

Anyway the field named as 'isSomething' should be boolean.